library/src/main/java/com/novoda/downloadmanager/MigrationFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.downloadmanager;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.io.File;
+
+public final class MigrationFactory {
+
+    private MigrationFactory() {
+        // Uses static methods.
+    }
+
+    public static Migrator createVersionOneToVersionTwoMigrator(Context context, File databasePath, Migrator.Callback callback) {
+        if (!databasePath.exists()) {
+            return new Migrator() {
```

Comment:
We could have a static NO_OP migrator defined in the interface

Reply:
2639fbe2a260b7f4511fd44f38d735946c569eff